game_eggs/steamcmd_servers/neosvr/egg-neos-v-r.json

```diff
@@ -0,0 +1,82 @@
+            "container": "ghcr.io\/pterodactyl\/installers:debian",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "Steam Account User",
+            "description": "The username for your Steam account.",
+            "env_variable": "STEAM_USER",
+            "default_value": "null",
```

Comment:
a default value is not null it is either empty or filled in. same for all variables